velox/substrait/tests/VeloxSubstraitRoundTripTest.cpp

```diff
@@ -441,6 +443,21 @@ TEST_F(VeloxSubstraitRoundTripTest, arrayLiteral) {
       "array[], array[array[1,2,3], array[4,5]]");
 }
 
+TEST_F(VeloxSubstraitRoundTripTest, dateType) {
+  auto a = makeFlatVector<int32_t>({0, 1});
+  auto b = makeFlatVector<double_t>({0.905791934145, 0.968867771124});
```

Comment:
Do we have to use these cryptic values? Can we use 0.1, 3.4 or similar instead?

Reply:
I have the same feeling. I merely assumed the existing test cases and used that data.

Reply:
Updated with simpler values. :slightly_smiling_face: